devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/QuarkusPlugin.java

```diff
@@ -423,14 +424,22 @@ private static void configureGenerateCodeTask(QuarkusGenerateCode task, String g
         task.getGeneratedOutputDirectory().set(generatedSources.getJava().getClassesDirectory().get().getAsFile());
     }
 
+    private void createSourceSets(Project project) {
```

Comment:
Where were the sourcesets being created before this change?

Reply:
Can we remove it from there?

Reply:
The sourcesets creation is moved from `registerTasks` to `createSourceSets`, any `sourceSets.create` in `registerTasks` is now replaced with `sourceSets.getByName`. Or am I missing something?

Reply:
Oh I understand now. Got it. :)